crowbar_framework/app/views/barclamp/neutron/_edit_attributes.html.haml

```diff
@@ -14,8 +14,11 @@
 
       = select_field :networking_plugin, :collection => :networking_plugins_for_neutron
 
-      #mode_container
-        = select_field :networking_mode, :collection => :networking_modes_for_neutron
+      #ml2_mechanism_drivers_container
+        = string_field %w(ml2_mechanism_drivers)
```

Comment:
I'm mostly worried about two things: people not knowing what they can enter in the field (so making this useless; could be fixed with a help label), and people putting wrong drivers in this field (can be fixed with validation).